### DISCUSSION TITLE
Print Temp Request Issue

### DISCUSSION THREAD

**Discussion by Methematics-LauncherPope** [Guilouz/Creality-Helper-Script-Wiki#542] [Issues]
Been having this issue since rooting my K1 MAX awhile ago. Can't quite seem to find the issue. When the print starts it will do the usual print start sequence but instead of heating up to the desired first layer temp, it will go to the temp set in PRINTER_PARAM.cfg under a [custom_macro]. My PRINT_START macro does not seem to be sending the desired temps although it does include M109 and M190.

Any advice would be greatly appreciated. I've been stumped on this for weeks.

**Comment by Methematics-LauncherPope** [comment]
I just noticed there was a big change that requires factory reset. I will start with a fresh reset and re-root the printer.

**Comment by Guilouz** [comment]
Duplicate
